config/crd/bases/delivery.ocm.software_componentversions.yaml

```diff
@@ -48,40 +48,39 @@ spec:
           metadata:
             type: object
           spec:
-            description: ComponentVersionSpec defines the desired state of ComponentVersion
+            description: ComponentVersionSpec specifies the configuration required
+              to retrieve a component descriptor for a component version.
             properties:
               component:
-                description: 'Every Component Version has a name. Name and version
-                  are the identifier for a Component Version and therefor for the
-                  artifact set described by it. A component name SHOULD reference
-                  a location where the component’s resources (typically source code,
-                  and/or documentation) are hosted. It MUST be a DNS compliant name
-                  with lowercase characters and MUST contain a name after the domain.
-                  Examples: - github.com/pathToYourRepo'
+                description: Component specifies the name of the ComponentVersion.
                 type: string
               interval:
+                description: Interval specifies the interval at which the Repository
+                  will be checked for updates.
                 type: string
               references:
+                description: References specifies configuration for the handling of
+                  nested component references.
                 properties:
                   expand:
                     type: boolean
                 type: object
               repository:
-                description: Repository defines the OCM Repository.
+                description: Repository provides details about the OCI repository
+                  from which the component descriptor can be retrieved.
                 properties:
                   secretRef:
-                    description: LocalObjectReference contains enough information
-                      to let you locate the referenced object inside the same namespace.
+                    description: SecretRef specifies the credentials used to access
+                      the OCI registry.
                     properties:
                       name:
                         description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
                           TODO: Add other useful fields. apiVersion, kind, uid?'
                         type: string
                     type: object
                   url:
-                    description: 'TODO@souleb: do we need a scheme for the url? add
-                      description for each field Do we need a type field? (e.g. oci,
-                      git, s3, etc.)'
+                    description: URL specifies the URL of an OCI repository in which
```

Comment:
Dont we refer the registry URL here rather than the repository? the repository url is added later in the controller by the oci-cli methods